src/components/NotePopup/NotePopup.js

```diff
@@ -83,12 +88,12 @@ const NotePopup = ({ annotation, setIsEditing }) => {
       </div>
       {isOpen && (
         <div className="options" onClick={closePopup}>
-          {!isEditDisabled && (
+          {isEditable && (
```

Comment:
Sorry, not sure if I missed something, but can't we have `!isEditDisabled && canModifyContents && (...)`?
Why do we need `isEditable` and `isDeletable`?

```javascript
return !canModify || isDisabled ? null : (
  <div
    className="NotePopup"
    data-element="notePopup"
    onMouseDown={e => e.stopPropagation()}
  >
    <div className="overflow" onClick={togglePopup}>
      <Icon glyph="ic_overflow_black_24px" />
    </div>
    {isOpen && (
      <div className="options" onClick={closePopup}>
        {!isEditDisabled && canModifyContents && (
          <div data-element="notePopupEdit" onClick={handleEdit}>
            {t('action.edit')}
          </div>
        )}
        {!isDeleteDisabled && (
          <div data-element="notePopupDelete" onClick={handleDelete}>
            {t('action.delete')}
          </div>
        )}
      </div>
    )}
  </div>
```

Reply:
You mean do something like 
<pre>
 return !(  <b>!(isEditDisabled && canModifyContents) || !(isDeleteDisabled && canModify)</b> ) || isDisabled ? null : (
    < div
      className="NotePopup"
      data-element="notePopup"
      onMouseDown={e => e.stopPropagation()}
    >
      <  div className="overflow" onClick={togglePopup}>
        <Icon glyph="ic_overflow_black_24px" />
      </  div>
      {isOpen && (
        <  div className="options" onClick={closePopup}>
          {  <b> isEditDisabled && canModifyContents </b> && (
            <  div data-element="notePopupEdit" onClick={handleEdit}>
              {t('action.edit')}
            </  div>
          )}
          {  <b> isDeleteDisabled && canModify </b> && (
            <  div data-element="notePopupDelete" onClick={handleDelete}>
              {t('action.delete')}
            </  div>
          )}
        </  div>
      )}
    </div>
  );
</pre>

Reply:
Nah I just meant the code in my response.
Is just changing 
```javascript
        {!isEditDisabled && (
          <div data-element="notePopupEdit" onClick={handleEdit}>
            {t('action.edit')}
          </div>
        )}
```
to
```javascript
        {!isEditDisabled && canModifyContents && (
          <div data-element="notePopupEdit" onClick={handleEdit}>
            {t('action.edit')}
          </div>
        )}
```
wrong?

Reply:
Why do we want to change `!canModify || isDisabled ?` to something else?

Reply:
"isEditable" is made up of  "!isEditDisabled && canModifyContents" already.  I guess the name is similar to "isEditDisabled" (which is when the dataElement is disabled)

Reply:
I'm using isEditable and isDeletable as temp variables because we are also replacing 
```javaScript
 return !canModify || isDisabled ? null : (
```
with 
```javaScript
 return !(isEditable || isDeletable) || isDisabled ? null : (
```

Reply:
yeah this is the part I don't understand. I'm ok with having the `isEditable` variable, but I don't understand why we are replacing `!canModify` with `!(isEditable || isDeletable)`. Apparently they are not equivalent and I'm not sure if I'm missing something here.

Reply:
Actually let have a discussion about this later. I think the legacy UI comment section (above) might be doing something incorrectly, I want to confirm what the proper behavior it is for us to handle this.